projects/vdk-control-cli/src/vdk/internal/control/job/job_config.py

```diff
@@ -27,7 +31,22 @@ def __init__(self, data_job_path):
                 "Make sure the file is created "
                 "or double check the data job path is passed correctly.",
             )
-        self._config_ini.read(self._config_file)
+        try:
+            self._config_ini.read(self._config_file)
+        except (configparser.MissingSectionHeaderError, Exception) as e:
+            log.debug(e, exc_info=log.getEffectiveLevel() == logging.DEBUG)
+            raise VDKException(
```

Comment:
you can create helper method to open config.ini instead of duplicating the error handling. 
 
 We should try to follow https://en.wikipedia.org/wiki/Don%27t_repeat_yourself when possible

Reply:
Agreed. Fixed, thanks